remoteappmanager/cli/remoteappdb/__main__.py

```diff
@@ -0,0 +1,247 @@
+            apps = ["{} {} {} {}".format(app.image,
+                                         policy.volume_source,
+                                         policy.volume_target,
+                                         policy.volume_mode)
+                    for app, policy in orm.apps_for_user(session, user)]
+
+            print("{}:{} | {} | {}".format(
+                user.id,
+                user.name,
+                ",".join(teams),
+                ",".join(apps)
+            ))
+
+
+# -------------------------------------------------------------------------
+# Team commands
+
+
+@cli.group()
+def team():
+    """Subcommand to manage teams."""
+    pass
+
+
+@team.command()  # noqa
+@click.argument("team")
+@click.pass_context
+def create(ctx, team):
+    """Creates a new team TEAM."""
+    db = ctx.obj["db"]
+    session = db.create_session()
+    orm_team = orm.Team(name=team)
+    with orm.transaction(session):
+        session.add(orm_team)
+
+    print(orm_team.id)
+
+
+@team.command()  # noqa
+@click.pass_context
+def list(ctx):
+    """Show the current teams."""
+    db = ctx.obj["db"]
+    session = db.create_session()
+    with orm.transaction(session):
+        for team in session.query(orm.Team).all():
+            print("{}:{}".format(team.id, team.name))
+
+
+@team.command()
+@click.argument("user")
+@click.argument("team")
+@click.pass_context
+def adduser(ctx, user, team):
+    """Add a user USER to a team TEAM."""
+    db = ctx.obj["db"]
+    session = db.create_session()
+    with orm.transaction(session):
+        orm_team = session.query(orm.Team).filter(
+            orm.Team.name == team).first()
+        orm_user = session.query(orm.User).filter(
+            orm.User.name == user).first()
+        orm_team.users.append(orm_user)
+
+# -------------------------------------------------------------------------
+# App commands
+
+
+@cli.group()
+def app():
+    """Subcommand to manage applications."""
+    pass
+
+
+@app.command()  # noqa
+@click.argument("image")
+@click.pass_context
+def create(ctx, image):
+    """Creates a new application for image IMAGE."""
+    db = ctx.obj["db"]
+    session = db.create_session()
+    with orm.transaction(session):
+        orm_app = orm.Application(image=image)
+        session.add(orm_app)
+
+    print(orm_app.id)
+
+
+@app.command()  # noqa
+@click.pass_context
+def list(ctx):
+    """List all registered applications."""
+    db = ctx.obj["db"]
+    session = db.create_session()
+    with orm.transaction(session):
+        for orm_app in session.query(orm.Application).all():
+            print("{}:{}".format(orm_app.id, orm_app.image))
+
+
+@app.command()
+@click.argument("image")
+@click.argument("team")
+@click.option("--allow-home", type=click.BOOL, default=False)
+@click.option("--allow-team-view", type=click.BOOL, default=False)
+@click.option("--volume", type=click.STRING)
+@click.pass_context
+def expose(ctx, image, team, db, allow_home, allow_team_view, volume):
```

Comment:
`db` is taken from context, so it is not required here as a positional argument.